lib/process.js

```diff
@@ -25,7 +25,12 @@ exports.process = async ({ record, log, db, dynamo, ilp }) => {
   // Payout amount is in millicents (1/1000 of a cent)
   const payoutAmount = payouts.reduce((sum, p) => sum + p.amount, 0)
 
-  await ilp.sendIlpPayment({ pointer: ilpPaymentPointer, amount: payoutAmount })
+  const { success, remainingAmount } = await ilp.sendIlpPayment({ pointer: ilpPaymentPointer, amount: payoutAmount })
+  // If sending money for some reason didn't send the whole amount, mark this payout as paid and add a new
+  // payout that is the remaining amount
+  if (!success && remainingAmount > 0) {
+    await db.addDifferentialPayout({ maintainerId, remainingAmount })
+  }
```

Comment:
yeah here is where i'm thinking we do something like `if (!success) logError(); if (remaining amount) dbCall()`. 

wondering -- do we want to make ILP errors noisier ? thinking if we just log it we'll never actually know anything happened. maybe we update the DB and then throw so the lambda dies noisy ? idk, haven't thought it all the way through.

Reply:
I'm going to throw after writing to DB, like you mentioned. if that happens too frequently and it's recovering just fine, we can remove the thrower.